@Raimondi `newsboat` 2.29 is failing to build with:

```
  make: *** No rule to make target `src/configcontainer.o', needed by `libboat.a'.  Stop.
```

--- a/net/newsboat/Portfile
+++ b/net/newsboat/Portfile
@@ -5,7 +5,7 @@ PortGroup           rust        1.0
 PortGroup           github      1.0
 PortGroup           makefile    1.0
 
-github.setup        newsboat newsboat 2.28 r
+github.setup        newsboat newsboat 2.29 r
 github.tarball_from archive
 revision            0
 
@@ -24,9 +24,9 @@ maintainers         {en.sent.com:macports @Raimondi} \
                     openmaintainer
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  f417257ecda8df46d7e2203821e6f162953fc2fd \
-                    sha256  72d072752292f7433490a1f78e0c4fc7bec248592132b150e924637e0647caaa \
-                    size    1149282 \
+                    rmd160  a2a6ab97e684efd6ecfaeee79081d6c2e2663717 \
+                    sha256  610200d3a621411014afaab3af0fed29abb3e0cd19138e74f03a4d6d66f9d0d9 \
+                    size    1158514 \
 
 compiler.cxx_standard       2011
 openssl.branch              3
@@ -40,17 +40,22 @@ build.env-append            GETTEXT_DIR=${prefix}
 
 build.pre_args-append       prefix=${prefix}
 
+build.cmd                   ${prefix}/bin/gmake
+
 build.args-append           BUILD_TYPE="release" \
                             CARGO_BUILD_FLAGS="-v --frozen --release" \
                             PROFILE=
 
 destroot.pre_args-append    prefix=${prefix}
 
+destroot.cmd                ${prefix}/bin/gmake
+
 use_configure               yes
 
 depends_build-append        port:asciidoctor \
                             port:docbook-xsl-nons \
                             port:gettext \
+                            port:gmake \
                             port:pkgconfig
 
 depends_lib-append          port:curl \
@@ -82,92 +87,103 @@ cargo.crates \
     addr2line                       0.17.0  b9ecd88a8c8378ca913a680cd98f0f13ac67383d35993f86c90a70e3f137816b \
     adler                            1.0.2  f26201604c87b1e01bd3d98f8d5d9a8fcbb815e8cedb41ffccbeb4bf593a35fe \
     aho-corasick                    0.7.18  1e37cfd5e7657ada45f742d6e99ca5788580b5c529dc78faf11ece6dc702656f \
+    android_system_properties        0.1.5  819e7219dbd41043ac279b19830f2efc897156490d7fd6ea916720117ee66311 \
     autocfg                          1.1.0  d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa \
-    backtrace                       0.3.64  5e121dee8023ce33ab248d9ce1493df03c3b38a659b240096fcbd7048ff9c31f \
+    backtrace                       0.3.65  11a17d453482a265fd5f8479f2a3f405566e6ca627837aaddb85af8b1ab8ef61 \
     bit-set                          0.5.2  6e11e16035ea35e4e5997b393eacbf6f63983188f7a2ad25bfb13465f5ad59de \
     bit-vec                          0.6.3  349f9b6a179ed607305526ca489b34ad0a41aed5f7980fa90eb03160b69598fb \
     bitflags                         1.3.2  bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a \
     block                            0.1.6  0d8c1fef690941d3e7788d328517591fecc684c084084702d6ff1641e993699a \
+    bumpalo                         3.10.0  37ccbd214614c6783386c1af30caf03192f17891059cecc394b4fb119e363de3 \
     byteorder                        1.4.3  14c189c53d098945499cdfa7ecc63567cf3886b3332b312a5b4585d8d3a6a610 \
     cc                              1.0.73  2fff2a6927b3bb87f9595d67196a70493f627687a71d87a0d692242c33f58c11 \
     cfg-if                           1.0.0  baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd \
-    chrono                          0.4.19  670ad68c9088c2a963aaa298cb369688cf3f9465ce5e2d4ca10e6e0098a1ce73 \
+    chrono                          0.4.22  bfd4d1b31faaa3a89d7934dbded3111da0d2ef28e3ebccdb4f0179f5929d1ef1 \
     codespan-reporting              0.11.1  3538270d33cc669650c4b093848450d380def10c331d38c768e34cac80576e6e \
-    curl-sys            0.4.55+curl-7.83.1  23734ec77368ec583c2e61dd3f0b0e5c98b93abe6d2a004ca06b91dd7e3e2762 \
-    cxx                             1.0.68  7e599641dff337570f6aa9c304ecca92341d30bf72e1c50287869ed6a36615a6 \
-    cxx-build                       1.0.68  60e2434bc22249c056e12d2e87db46380730da0f2648471edea3e8e11834a892 \
-    cxxbridge-flags                 1.0.68  3894ad0c6d517cb5a4ce8ec20b37cd0ea31b480fe582a104c5db67ae21270853 \
-    cxxbridge-macro                 1.0.68  34fa7e395dc1c001083c7eed28c8f0f0b5a225610f3b6284675f444af6fab86b \
+    core-foundation-sys              0.8.3  5827cebf4670468b8772dd191856768aedcb1b0278a04f989f7766351917b9dc \
+    curl-sys            0.4.57+curl-7.85.0  c2f5c209fdc3b856c446c52a1f9e90db20ea2b1bbbbd60bc18239174fa6eae70 \
+    cxx                             1.0.76  fd0492b760d2a9b9c5d1e2132db2dcbba07a2045df6f14694b136eec5b2053ef \
+    cxx-build                       1.0.76  fc65e68d05a2bf20230ea07fff3f26aaa15d057d8361438859f0fe66a89ce288 \
+    cxxbridge-flags                 1.0.76  2ebfd09b0a600cf97b53c16eec6591893cbcaffe3f98c17c4e89b3b3a672c33a \
+    cxxbridge-macro                 1.0.76  f1ccac323f5afe0678c7129b8346c2be2d2213d8c58507facfb4987d109abf1b \
     dirs                             4.0.0  ca3aa72a6f96ea37bbc5aa912f6788242832f75369bdfdadcb0e38423f100059 \
     dirs-sys                         0.3.7  1b1d1d91c932ef41c0f2663aa8b0ca0342d444d842c06914aa0a7e352d0bada6 \
-    fastrand                         1.7.0  c3fcf0cee53519c866c09b5de1f6c56ff9d647101f81c1964fa632e148896cdf \
+    fastrand                         1.8.0  a7a407cfaa3385c4ae6b23e84623d48c2798d06e3e6a1878f7f59f17b3f86499 \
     fnv                              1.0.7  3f9eec918d3f24069decb9af1554cad7c880e2da24a9afd88aca000531ab82c1 \
-    form_urlencoded                  1.0.1  5fc25a87fa4fd2094bffb06925852034d90a17f0d1e05197d4956d3555752191 \
-    getrandom                        0.2.5  d39cd93900197114fa1fcb7ae84ca742095eed9442088988ae74fa744e930e77 \
+    form_urlencoded                  1.1.0  a9c384f161156f5260c24a097c56119f9be8c798586aecc13afbcbe7b7e26bf8 \
+    getrandom                        0.2.7  4eb1a864a501629691edf6c15a593b7a51eebaa1e8468e9ddc623de7c9b58ec6 \
     gettext-rs                       0.7.0  e49ea8a8fad198aaa1f9655a2524b64b70eb06b2f3ff37da407566c93054f364 \
     gettext-sys                     0.21.3  c63ce2e00f56a206778276704bbe38564c8695249fdc8f354b4ef71c57c3839d \
     gimli                           0.26.1  78cc372d058dcf6d5ecd98510e7fbc9e5aec4d21de70f65fea8fecebcd881bd4 \
-    idna                             0.2.3  418a0a6fab821475f634efe3ccc45c013f742efe03d853e8d3355d5cb850ecf8 \
+    iana-time-zone                  0.1.47  4c495f162af0bf17656d0014a0eded5f3cd2f365fdd204548c2869db89359dc7 \
+    idna                             0.3.0  e14ddfc70884202db2244c223200c204c2bda1bc6e0998d11b5e024d657209e6 \
     instant                         0.1.12  7a5bbe824c507c5da5956355e86a746d82e0e1464f65d862cc5e71da70e94b2c \
+    js-sys                          0.3.59  258451ab10b34f8af53416d1fdab72c22e805f0c92a1136d59470ec0b11138b2 \
     lazy_static                      1.4.0  e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646 \
-    lexopt                           0.2.0  de7513aea7d10dc0694d719ac53de3b74f1600e41b9f81ed9f395d8ec36a2a70 \
-    libc                           0.2.126  349d5a591cd28b49e1d1037471617a32ddcda5731b99419008085f72d5a53836 \
-    libz-sys                         1.1.5  6f35facd4a5673cb5a48822be2be1d4236c1c99cb4113cab7061ac720d5bf859 \
+    lexopt                           0.2.1  478ee9e62aaeaf5b140bd4138753d1f109765488581444218d3ddda43234f3e8 \
+    libc                           0.2.133  c0f80d65747a3e43d1596c7c5492d95d5edddaabd45a7fcdb02b95f644164966 \
+    libz-sys                         1.1.8  9702761c3935f8cc2f101793272e202c72b99da8f4224a19ddcf1279a6450bbf \
     link-cplusplus                   1.0.6  f8cae2cd7ba2f3f63938b9c724475dfb7b9861b545a90324476324ed21dbc8c8 \
     locale_config                    0.3.0  08d2c35b16f4483f6c26f0e4e9550717a2f6575bcd6f12a53ff0c490a94a6934 \
+    log                             0.4.17  abb12e687cfb44aa40f41fc3978ef76448f9b6038cad6aef4259d3c095a2382e \
     malloc_buf                       0.0.6  62bb907fe88d54d8d9ce32a3cceab4218ed2f6b7d35617cafe9adf84e43919cb \
-    matches                          0.1.9  a3e378b66a060d48947b590737b30a1be76706c8dd7b8ba0f2fe3989c68a853f \
     md5                              0.7.0  490cc448043f947bae3cbee9c203358d62dbee0db12107a74be5c30ccfd09771 \
-    memchr                           2.4.1  308cc39be01b73d0d18f82a0e7b2a3df85245f84af96fdddc5d202d27e47b86a \
+    memchr                           2.5.0  2dffe52ecf27772e601905b7522cb4ef790d2cc203488bbd0e2fe85fcb74566d \
     minimal-lexical                  0.2.1  68354c5c6bd36d73ff3feceb05efa59b6acb7626617f4962be322a825e61f79a \
-    miniz_oxide                      0.4.4  a92518e98c078586bc6c934028adcca4c92a53d6a958196de835170a01d84e4b \
+    miniz_oxide                      0.5.3  6f5c75688da582b8ffc1f1799e9db273f32133c49e048f614d22ec3256773ccc \
     natord                           1.0.9  308d96db8debc727c3fd9744aac51751243420e46edf401010908da7f8d5e57c \
     nom                              7.1.1  a8903e5a29a317527874d0402f867152a3d21c908bb0b933e416c65e301d4c36 \
-    num-integer                     0.1.44  d2cc698a63b549a70bc047073d2949cce27cd1c7b0a4a862d08a8031bc2801db \
-    num-traits                      0.2.14  9a64b1ec5cda2586e284722486d802acf1f7dbdc623e2bfc57e65ca1cd099290 \
+    num-integer                     0.1.45  225d3389fb3509a24c93f5c29eb6bde2586b98d9f016636dff58d7c6f7569cd9 \
+    num-traits                      0.2.15  578ede34cf02f8924ab9447f50c28075b4d3e5b269972345e7e0372b38c6cdcd \
     objc                             0.2.7  915b1b472bc21c53464d6c8461c9d3af805ba1ef837e1cac254428f4a77177b1 \
     objc-foundation                  0.1.1  1add1b659e36c9607c7aab864a76c7a4c2760cd0cd2e120f3fb8b952c7e22bf9 \
     objc_id                          0.1.1  c92d4ddb4bd7b50d730c215ff871754d0da6b2178849f8a2a2ab69712d0c073b \
-    object                          0.27.1  67ac1d3f9a1d3616fd9a60c8d74296f22406a238b6a72f5cc1e6f314df4ffbf9 \
-    once_cell                       1.12.0  7709cef83f0c1f58f666e746a08b21e0085f7440fa6a29cc194d68aac97a4225 \
-    percent-encoding                 2.1.0  d4fd5641d01c8f18a23da7b6fe29298ff4b55afcccdf78973b24cf3175fee32e \
-    pkg-config                      0.3.24  58893f751c9b0412871a09abd62ecd2a00298c6c83befa223ef98c52aef40cbe \
+    object                          0.28.4  e42c982f2d955fac81dd7e1d0e1426a7d702acd9c98d19ab01083a6a0328c424 \
+    once_cell                       1.15.0  e82dad04139b71a90c080c8463fe0dc7902db5192d939bd0950f074d014339e1 \
+    percent-encoding                 2.2.0  478c572c3d73181ff3c2539045f6eb99e5491218eae919370993b890cdbdd98e \
+    pkg-config                      0.3.25  1df8c4ec4b0627e53bdf214615ad287367e482558cf84b109250b37464dc03ae \
     ppv-lite86                      0.2.16  eb9f9e6e233e5c4a35559a617bf40a4ec447db2e84c20b55a6f83167b7e57872 \
-    proc-macro2                     1.0.36  c7342d5883fbccae1cc37a2353b09c87c9b0f3afd73f5fb9bba687a1f733b029 \
+    proc-macro2                     1.0.40  dd96a1e8ed2596c337f8eae5f24924ec83f5ad5ab21ea8e455d3566c69fbcaf7 \
     proptest                         1.0.0  1e0d9cc07f18492d879586c92b485def06bc850da3118075cd45d50e9c95b0e5 \
     quick-error                      1.2.3  a1d01941d82fa2ab50be1e79e6714289dd7cde78eba4c074bc5a4374f650dfe0 \
     quick-error                      2.0.1  a993555f31e5a609f617c12db6250dedcac1b0a85076912c436e6fc9b2c8e6a3 \
-    quote                           1.0.16  b4af2ec4714533fcdf07e886f17025ace8b997b9ce51204ee69b6da831c3da57 \
+    quote                           1.0.20  3bcdf212e9776fbcb2d23ab029360416bb1706b1aea2d1a5ba002727cbcab804 \
     rand                             0.8.5  34af8d1a0e25924bc5b7c43c079c942339d8f0a8b57c39049bef581b46327404 \
     rand_chacha                      0.3.1  e6c10a63a0fa32252be49d21e7709d4d4baf8d231c2dbce1eaa8141b9b127d88 \
     rand_core                        0.6.3  d34f1408f55294453790c48b2f1ebbb1c5b4b7563eb1f418bcfcfdbb06ebb4e7 \
     rand_xorshift                    0.3.0  d25bf25ec5ae4a3f1b92f929810509a2f53d7dca2f50b794ff57e3face536c8f \
-    redox_syscall                   0.2.11  8380fe0152551244f0747b1bf41737e0f8a74f97a14ccefd1148187271634f3c \
-    redox_users                      0.4.2  7776223e2696f1aa4c6b0170e83212f47296a00424305117d013dfe86fb0fe55 \
-    regex                            1.5.5  1a11647b6b25ff05a515cb92c365cec08801e83423a235b51e231e1808747286 \
-    regex-syntax                    0.6.25  f497285884f3fcff424ffc933e56d7cbca511def0c9831a7f9b5f6153e3cc89b \
+    redox_syscall                   0.2.13  62f25bc4c7e55e0b0b7a1d43fb893f4fa1361d0abe38b9ce4f323c2adfe6ef42 \
+    redox_users                      0.4.3  b033d837a7cf162d7993aded9304e30a83213c648b6e389db233191f891e5c2b \
+    regex                            1.5.6  d83f127d94bdbcda4c8cc2e50f6f84f4b611f69c902699ca385a39c3a75f9ff1 \
+    regex-syntax                    0.6.26  49b3de9ec5dc0a3417da371aab17d729997c15010e7fd24ff707773a33bddb64 \
     remove_dir_all                   0.5.3  3acd125665422973a33ac9d3dd2df85edad0f4ae9b00dafb1a05e43a9f5ef8e7 \
     rustc-demangle                  0.1.21  7ef03e0a2b150c7a90d01faf6254c9c48a41e95fb2a8c2ac1c6f0d2b9aefc342 \
     rusty-fork                       0.3.0  cb3dcc6e454c328bb824492db107ab7c0ae8fcffe4ad210136ef014458c1bc4f \
     scratch                          1.0.1  96311ef4a16462c757bb6a39152c40f58f31cd2602a40fceb937e2bc34e6cbab \
     section_testing                  0.0.5  5fd2493b68af689f4863060b240cbdffb350cee9ed69e2c50f8d71a62ca2aea1 \
-    syn                             1.0.89  ea297be220d52398dcc07ce15a209fce436d361735ac1db700cab3b6cdfb9f54 \
+    syn                             1.0.98  c50aef8a904de4c23c788f104b7dddc7d6f79c647c7c8ce4cc8f73eb0ca773dd \
     temp-dir                        0.1.11  af547b166dd1ea4b472165569fc456cfb6818116f854690b0ff205e636523dab \
     tempfile                         3.3.0  5cdb1ef4eaeeaddc8fbd371e5017057064af0911902ef36b39801f67cc6d79e4 \
     termcolor                        1.1.3  bab24d30b911b2376f3a13cc2cd443142f0c81dda04c118693e35b3835757755 \
-    thiserror                       1.0.30  854babe52e4df1653706b98fcfc05843010039b406875930a70e4d9644e5c417 \
-    thiserror-impl                  1.0.30  aa32fd3f627f367fe16f893e2597ae3c05020f8bba2666a4e6ea73d377e5714b \
-    time                            0.1.43  ca8a50ef2360fbd1eeb0ecd46795a87a19024eb4b53c5dc916ca1fd95fe62438 \
-    tinyvec                          1.5.1  2c1c1d5a42b6245520c249549ec267180beaffcc0615401ac8e31853d4b6d8d2 \
+    thiserror                       1.0.31  bd829fe32373d27f76265620b5309d0340cb8550f523c1dda251d6298069069a \
+    thiserror-impl                  1.0.31  0396bc89e626244658bef819e22d0cc459e795a5ebe878e6ec336d1674a8d79a \
+    time                            0.1.44  6db9e6914ab8b1ae1c260a4ae7a49b6c5611b40328a735b21862567685e73255 \
+    tinyvec                          1.6.0  87cc5ceb3875bb20c2890005a4e226a4651264a5c75edb2421b52861a0a0cb50 \
     tinyvec_macros                   0.1.0  cda74da7e1a664f795bb1f8a87ec406fb89a02522cf6e50620d016add6dbbf5c \
-    unicode-bidi                     0.3.7  1a01404663e3db436ed2746d9fefef640d868edae3cceb81c3b8d5732fda678f \
-    unicode-normalization           0.1.19  d54590932941a9e9266f0832deed84ebe1bf2e4c9e4a3554d393d18f5e854bf9 \
-    unicode-width                    0.1.9  3ed742d4ea2bd1176e236172c8429aaf54486e7ac098db29ffe6529e0ce50973 \
-    unicode-xid                      0.2.2  8ccb82d61f80a663efe1f787a51b16b5a51e3314d6ac365b08639f52387b33f3 \
-    url                              2.2.2  a507c383b2d33b5fc35d1861e77e6b383d158b2da5e14fe51b83dfedf6fd578c \
+    unicode-bidi                     0.3.8  099b7128301d285f79ddd55b9a83d5e6b9e97c92e0ea0daebee7263e932de992 \
+    unicode-ident                    1.0.1  5bd2fe26506023ed7b5e1e315add59d6f584c621d037f9368fea9cfb988f368c \
+    unicode-normalization           0.1.20  81dee68f85cab8cf68dec42158baf3a79a1cdc065a8b103025965d6ccb7f6cbd \
+    unicode-width                   0.1.10  c0edd1e5b14653f783770bce4a4dabb4a5108a5370a5f5d8cfe8710c361f6c8b \
+    url                              2.3.1  0d68c799ae75762b8c3fe375feb6600ef5602c883c5d21eb51c09f22b83c4643 \
     vcpkg                           0.2.15  accd4ea62f7bb7a82fe23066fb0957d48ef677f6eeb8215f372f52e48bb32426 \
     wait-timeout                     0.2.0  9f200f5b12eb75f8c1ed65abd4b2db8a6e1b138a20de009dacee265a2498f3f6 \
-    wasi     0.10.2+wasi-snapshot-preview1  fd6fbd9a79829dd1ad0cc20627bf1ed606756a7f77edff7b66b7064f9cb327c6 \
+    wasi     0.10.0+wasi-snapshot-preview1  1a143597ca7c7793eff794def352d41792a93c481eb1042423ff7ff72ba2c31f \
+    wasi     0.11.0+wasi-snapshot-preview1  9c8d87e72b64a3b4db28d11ce29237c246188f4f51057d65a7eab63b7987e423 \
+    wasm-bindgen                    0.2.82  fc7652e3f6c4706c8d9cd54832c4a4ccb9b5336e2c3bd154d5cccfbf1c1f5f7d \
+    wasm-bindgen-backend            0.2.82  662cd44805586bd52971b9586b1df85cdbbd9112e4ef4d8f41559c334dc6ac3f \
+    wasm-bindgen-macro              0.2.82  b260f13d3012071dfb1512849c033b1925038373aea48ced3012c09df952c602 \
+    wasm-bindgen-macro-support      0.2.82  5be8e654bdd9b79216c2929ab90721aa82faf65c48cdf08bdc4e7f51357b80da \
+    wasm-bindgen-shared             0.2.82  6598dd0bd3c7d51095ff6531a5b23e02acdc81804e30d8f07afb77b7215a140a \
     winapi                           0.3.9  5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419 \
     winapi-i686-pc-windows-gnu       0.4.0  ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6 \
     winapi-util                      0.1.5  70ec6ce85bb158151cae5e5c87f95a8e97d2c0c4b001223f33a334e3ce5de178 \

--- a/net/newsboat/files/patch-libiconv-makefile.diff
+++ b/net/newsboat/files/patch-libiconv-makefile.diff
@@ -1,26 +1,33 @@
---- Makefile	2021-07-25 14:29:31.000000000 -0400
-+++ Makefile	2021-07-25 14:30:20.000000000 -0400
-@@ -83,7 +83,7 @@
+--- Makefile	2022-09-27 15:17:22.000000000 -0400
++++ Makefile	2022-09-27 15:20:51.000000000 -0400
+@@ -56,7 +56,7 @@
  NEWSBOAT=newsboat
- NEWSBOAT_SOURCES:=$(shell cat mk/newsboat.deps)
- NEWSBOAT_OBJS:=$(patsubst %.cpp,%.o,$(NEWSBOAT_SOURCES))
+ NEWSBOAT_SRCS:=$(shell cat mk/newsboat.deps)
+ NEWSBOAT_OBJS:=$(patsubst %.cpp,%.o,$(NEWSBOAT_SRCS))
 -NEWSBOAT_LIBS=-lboat -lnewsboat -lfilter -lpthread -lrsspp -ldl
 +NEWSBOAT_LIBS=-lboat -lnewsboat -lfilter -lpthread -lrsspp -ldl -liconv
  
- RSSPPLIB_SOURCES=$(sort $(wildcard rss/*.cpp))
- RSSPPLIB_OBJS=$(patsubst rss/%.cpp,rss/%.o,$(RSSPPLIB_SOURCES))
-@@ -109,11 +109,11 @@
+ RSSPPLIB_SRCS=$(sort $(wildcard rss/*.cpp))
+ RSSPPLIB_OBJS=$(patsubst rss/%.cpp,rss/%.o,$(RSSPPLIB_SRCS))
+@@ -65,7 +65,7 @@
  PODBOAT=podboat
- PODBOAT_SOURCES:=$(shell cat mk/podboat.deps)
- PODBOAT_OBJS:=$(patsubst %.cpp,%.o,$(PODBOAT_SOURCES))
+ PODBOAT_SRCS:=$(shell cat mk/podboat.deps)
+ PODBOAT_OBJS:=$(patsubst %.cpp,%.o,$(PODBOAT_SRCS))
 -PODBOAT_LIBS=-lboat -lnewsboat -lfilter -lpthread -ldl
 +PODBOAT_LIBS=-lboat -lnewsboat -lfilter -lpthread -ldl -liconv
  
- ifeq (, $(filter Linux GNU GNU/%, $(shell uname -s)))
--NEWSBOAT_LIBS+=-liconv -lintl
--PODBOAT_LIBS+=-liconv -lintl
-+NEWSBOAT_LIBS+=-lintl
-+PODBOAT_LIBS+=-lintl
+ TEST_SRCS:=$(wildcard test/*.cpp test/test_helpers/*.cpp)
+ TEST_OBJS:=$(patsubst %.cpp,%.o,$(TEST_SRCS))
+@@ -103,10 +103,8 @@
+ LDFLAGS+=-L$(CARGO_TARGET_DIR)/$(BUILD_TYPE)
  endif
  
+-ifeq (, $(filter Linux GNU GNU/%, $(shell uname -s)))
+-NEWSBOAT_LIBS+=-liconv -lintl
+-PODBOAT_LIBS+=-liconv -lintl
+-endif
++NEWSBOAT_LIBS+=-lintl
++PODBOAT_LIBS+=-lintl
+ 
  # additional commands
+ MKDIR=mkdir -p

--- a/net/newsboat/files/patch-libiconv-rust.diff
+++ b/net/newsboat/files/patch-libiconv-rust.diff
@@ -1,6 +1,6 @@
---- rust/libnewsboat/src/utils.rs	2021-07-25 14:23:26.000000000 -0400
-+++ rust/libnewsboat/src/utils.rs	2021-07-25 14:25:52.000000000 -0400
-@@ -747,12 +747,15 @@
+--- rust/libnewsboat/src/utils.rs	2022-09-27 15:24:48.000000000 -0400
++++ rust/libnewsboat/src/utils.rs	2022-09-27 15:29:38.000000000 -0400
+@@ -824,12 +824,15 @@
  // On FreeBSD, link with GNU libiconv; the iconv implementation in libc doesn't support //TRANSLIT
  // and WCHAR_T. This is also why we change the symbol names from "iconv" to "libiconv" below.
  #[cfg_attr(target_os = "freebsd", link(name = "iconv"))]
@@ -16,7 +16,7 @@
      #[cfg_attr(target_os = "openbsd", link_name = "libiconv")]
      pub fn iconv(
          cd: iconv_t,
-@@ -762,6 +765,7 @@
+@@ -839,6 +842,7 @@
          outbytesleft: *mut size_t,
      ) -> size_t;
      #[cfg_attr(target_os = "freebsd", link_name = "libiconv_close")]

